app/vendors/shells/sphinx_conf.php

```diff
@@ -220,6 +220,67 @@ public function __construct(&$dispatch) {
             )
         );
 
+        $this->indexExtraOptions['lat'] =
+            "
+        charset_table = ".implode(', ', array_merge(
+            array(
+                'A..H->a..h', 'a..h',
+                'I->i', 'J->i', 'j->i', 'i',
+                'K..T->k..t', 'k..t',
+                'U->v', 'u->v', 'V->v', 'W->v', 'w->v', 'v',
+                'X..Z->x..z', 'x..z',
+                'U+100->a',       # cap a + macron -> small a
+                'U+101->a',       # small a + macron -> small a
+                'U+102->a',       # cap a + breve -> small a
+                'U+103->a',       # small a + breve -> small a
+                'U+C1->a',        # cap a + acute -> small a
+                'U+E1->a',        # small a + acute -> small a
+                'U+112->e',       # same pattern as for a
+                'U+113->e',
+                'U+114->e',
+                'U+115->e',
+                'U+C9->e',
+                'U+E9->e',
+                'U+12A->i',       # same pattern as for a
```

Comment:
What I meant here is that we follow the same pattern that we followed for "a", except that here we're following it for "e", for "i", and so on. Thus, the comment on line 239 would say "# cap e + macron -> small e", the comment on line 244 would say "# cap i + macron -> small i", and so on. Likewise, the comment on line 240 would say "# small e + macron -> small e", the comment on line 245 would say "# small i + macron -> small i", and so on.